cranelift-codegen/src/isa/x86/abi.rs

```diff
@@ -680,7 +723,8 @@ fn insert_common_prologue(
     reg_type: ir::types::Type,
     csrs: &RegisterSet,
     isa: &dyn TargetIsa,
-) {
+) -> Option<CFAState> {
+    let word_size = isa.pointer_bytes();
```

Comment:
Could this be converted as `isize` directly here? It seems all the users want to use it as such (or `i64` anyways).